Scripts/Restriction/ranacompiler.py

```diff
@@ -452,7 +454,10 @@ def build_dict(self):
             try:
                 enzyme_id = enzyme_id_dict[name]
             except KeyError:
-                print(f"Could not find REBASE enzyme ID for {name}: omitting")
+                # be more specific, this is a warning it just omits the REBASE URL
+                print(
+                    f"WARNING : Could not find REBASE enzyme ID for {name}: omitting url"
+                )
```

Comment:
This seems like a useful logging clarification

Reply:
Yes as per my comment before, this was needed on the output because the enzyme is added but just the URL is omitted, before it looked like "the enzyme" was omitted.